### PR TITLE
Fixed https://issues.jboss.org/browse/TEIID-4717

### PR DESCRIPTION
**Pull request by nmvijay**
@rareddy I have fixed the https://issues.jboss.org/browse/TEIID-4717 issue, please review and let me know if any issues

### CODE REVIEW

**Comment by rareddy** [comment]
If you do not mind can you remove all format specific changes and system.out in the commit? Also please remove the system.out. Other than that it looks good.

**Comment by nmvijay** [comment]
my mistake, removed the SOP and formatting. please check now.

BTW, do you have eclipse code formatter template for Teiid

File: connectors/cassandra/translator-cassandra/src/main/java/org/teiid/translator/cassandra/CassandraMetadataProcessor.java [hunk 1]
```diff
@@ -61,7 +69,14 @@ public void process(MetadataFactory factory, CassandraConnection connection) thr
 	private void addTable(MetadataFactory factory, TableMetadata columnFamily) {
 		Table table = factory.addTable(columnFamily.getName());
 		addColumnsToTable(factory, table, columnFamily);
-		addPrimaryKey(factory, table, columnFamily);
+        addPrimaryKey(factory, table, columnFamily);
```
Comment (rareddy) [inline connectors/cassandra/translator-cassandra/src/main/java/org/teiid/translator/cassandra/CassandraMetadataProcessor.java]: This change is from earlier commit, you need to rebase your commit from master, such that it only pulls in your changes

File: connectors/cassandra/translator-cassandra/src/main/java/org/teiid/translator/cassandra/CassandraMetadataProcessor.java [hunk 2]
```diff
@@ -89,26 +103,70 @@ private void addPrimaryKey(MetadataFactory factory, Table table, TableMetadata c
...
-				c.setSearchType(SearchType.Searchable);
-			}
-			else {
-				c.setSearchType(SearchType.Unsearchable);
-			}
+			c.setSearchType(SearchType.Unsearchable);
 		}
 	}
+
+    private String asTeiidRuntimeType(Name name) {
```
Comment (rareddy) [inline connectors/cassandra/translator-cassandra/src/main/java/org/teiid/translator/cassandra/CassandraMetadataProcessor.java]: Same as previous one, from earlier commit

File: connectors/cassandra/translator-cassandra/src/main/java/org/teiid/translator/cassandra/CassandraSQLVisitor.java [hunk 3]
```diff
@@ -42,6 +42,8 @@
 import org.teiid.translator.TypeFacility;
 
 public class CassandraSQLVisitor extends SQLStringVisitor {
+    
+    	private static final String ALLOW_FILTERING = "ALLOW FILTERING";
```
Comment (rareddy) [inline connectors/cassandra/translator-cassandra/src/main/java/org/teiid/translator/cassandra/CassandraSQLVisitor.java]: This looks good

**Pull request closed without merging**